csharp/ql/src/experimental/ir/implementation/unaliased_ssa/internal/SSAConstruction.qll

```diff
@@ -64,7 +64,7 @@ private module Cached {
     or
     instr = reusedPhiInstruction(_) and
     // Check that the phi instruction is *not* degenerate, but we can't use
-    // getDegeneratePhiOperand in the first stage with phi instyructions
+    // getDegeneratePhiOperand in the first stage with phi instructions
```

Comment:
equivalent to https://github.com/github/codeql/pull/10825/commits/73993edfd46b1b43240db1565fbc277241ad5949#diff-f5cf97073ed3726cc84eb446644c71394729d149e48b27d8a26ea7099a821d3bR67